ReactAndroid/src/main/java/com/facebook/react/views/viewpager/ReactViewPager.java

```diff
@@ -154,6 +158,7 @@ public boolean onTouchEvent(MotionEvent ev) {
     if (!mScrollEnabled) {
       return false;
     }
```

Comment:
That check is needed for both methods:
[onInterceptTouchEvent](https://github.com/kevinejohn/react-native/blob/1b6a51a81303d1698f86fce6f57f23b8b11b51b5/ReactAndroid/src/main/java/com/facebook/react/views/viewpager/ReactViewPager.java#L144)
[onTouchEvent](https://github.com/kevinejohn/react-native/blob/1b6a51a81303d1698f86fce6f57f23b8b11b51b5/ReactAndroid/src/main/java/com/facebook/react/views/viewpager/ReactViewPager.java#L157)